### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by knowbody** [knowbody/react-native-online-meetups#19]
Any feedback that you have, things that we can improve for the next meetup, please let us now here

**Comment by nikolenkoanton92** [comment]
@knowbody thank you for organize and thank you all speakers today, I am very enjoyed. Waiting for next meetup! 👍

**Comment by hypatiah** [comment]
@knowbody Was so helpful and awesome, thanks for putting this together. Definitely looking forward to the next one!

**Comment by franciscofsales** [comment]
Thanks guys, this is a much needed initiative and amazing. I'm tuned in for the next one 💯

**Comment by aubsw** [comment]
yeah it was definitely cool! discussion was great. would be really cool if we could get people from core react native team on there

**Comment by knowbody** [comment]
@aubreywahl thanks for suggestion I try to get someone for the next one :)

**Comment by knowbody** [comment]
anyone in particular from fb team?

**Comment by kelset** [comment]
I'm not from the fb team but here are my two cents ;)
- Parashuram and Mateusz have been really good hosts, keeping the conversation open and being "the voice" of us spectators
- I didn't mind going over the hour mark, since it was more of a conversation than simply having a list of mini-conferences one after the other
- screen sharing code from the IDE was not ideal, mostly because of readibility
- YouTube Live chat and SliDo are a bit redundant, I had the feeling that most watchers weren't using the latter (still thanks to our hosts no question went missing)

That said, overall it was a really good meetup: I had the feeling of being among friends in front of a beer, almost.

A couple of ideas for the future:
- Maybe, instead of Sli.do, a Twitter hashtag may be a good idea: it would still be "redundant" over the chat, but at least (since most of us use it already) it would give the possibility to tag the speaker whose the question is for. That said #reactnativeonlinemeetup is not usable, I mean. Really too long.
- I'm probably not the only one who'd like this "format" to be frequent over time, let's say every 2/3 weeks. The issue in this case would be that, keeping the format of having different people presenting something each time, maybe in 3/4 appointments there would be no more speakers. So I have this hypothesis to present you: 4/5 people each time, 2 of which have to "present" something (like the guys did this time) and then the "host" will present an argument, or question, or something (which may be decided starting from the list of Suggested Topics) and have the speakers talk around that for some time. The ideal situation would be, in that case, that the "topic" is decided like at least one week before so that people can start write questions around it that the speakers will try to answer (a way to do this could be to have a dedicated Issue here on GitHub for every meetup). Going for some version of this format would allow to have people which can propose to be on the "panel" without having to present something (but they feel confident about the topic so they want to be there to talk about it) and having people who can be on the panel more than one time.

(ok, sorry for the huge wall of text - feel free to smash those ideas down ;)  )

**Comment by rnagella** [comment]
Under the past meetup along with the video link, we should write a synopsis of the talk. This will help user to choose if he wants to go and view specific discussion/s.

**Comment by knowbody** [comment]
@kelset thanks for the great feedback, I definitely take it to the account and apply it for the next one :) I also want to do a few changes which you recommended so hopefully it will get better next time

**Comment by AndyIsaacson** [comment]
I found out about this a couple of days after the fact, and really enjoyed watching it.  It felt hugely informative, and something that I've been missing so far - especially since there aren't a lot of React-Native users in my geographical region.

Other than watching this repo, what's the best way to be notified of upcoming online meetups?

**Comment by knowbody** [comment]
@AndyIsaacson I think the best to be up to date is just clicking `Watch` the repo. Or I usually tweet about it RN Online Meetup too ([@matzatorski](https://twitter.com/matzatorski))
